fio_wrapper/trigger_fio.py

```diff
@@ -128,7 +128,7 @@ def _log_payload(self, directory, user, uuid, sample, fio_jobs_dict, fio_version
                     with open(directory + '/' + str(log_file_name), 'r') as log_file:
                         for log_line in log_file:
                             log_line_values = str(log_line).split(", ")
-                            if len(log_line_values) == 4:
+                            if len(log_line_values) == 5:
```

Comment:
Do we want to have some backwards compatibility or simply state we support fio-3.19 (for now)... Since this won'w work with earlier versions?

Reply:
I guess this would be required in case of executing Snafu out from the Ripsaw context. Is that what you're thinking?

Reply:
My thoughts exactly... We could capture the version via `fio --version` to determine the code path... Or simply document we only support X version... I am not against either.